libr/core/cmd_write.c

```diff
@@ -94,21 +94,22 @@ static void cmd_write_op (RCore *core, const char *input) {
 	ut8 *buf;
 	int len;
 	const char* help_msg[] = {
-		"Usage:","wo[asmdxoArl24]"," [hexpairs] @ addr[!bsize]",
+		"Usage:","wo[asmdxoArl24][y]"," [hexpairs] @ addr[!bsize]",
 		"wow"," [val]", "==  write looped value (alias for 'wb')",
 		"woa"," [val]", "+=  addition (f.ex: woa 0102)",
 		"wos"," [val]", "-=  substraction",
 		"wom"," [val]", "*=  multiply",
 		"wod"," [val]", "/=  divide",
-		"woe"," [from to] [step] [wsz=1]","..  create sequence",
 		"wox"," [val]","^=  xor  (f.ex: wox 0x90)",
 		"woo"," [val]","|=  or",
 		"woA"," [val]","&=  and",
-		"woR","","random bytes (alias for 'wr $b')",
 		"wor"," [val]", ">>= shift right",
 		"wol"," [val]","<<= shift left",
 		"wo2"," [val]","2=  2 byte endian swap",
 		"wo4"," [val]", "4=  4 byte endian swap",
+		"wo*y", "", "same as their counterparts above, but using clipboard instead of [val]",
```

Comment:
r2 have a huge codebase and we try to have as much testcases as possible in order to not break stuff, because one of the main premises is the continuous refactoring, mainly because most of the stuff that is implemented in r2 is done "quickly" because is a feature someone needs for something NOW. but there are always better ways to do stuff, so that's where the refactoring enters the game. you can do this alphabetically sorting thing in the same PR, there's no real issue in sorting out a text array that will be processed by humans.